perf/stability/README.md

```diff
@@ -4,6 +4,30 @@ This directory providestests that exercise various Istio features to ensure stab
 
 The intent of these tests is to be run continuously for extend periods of time, to ensure features are stable over long periods of time and in real world scenarios, which differentiates them from integration tests.
 
+## Run Release Qualification Test
+
+The long running test would deploy service graphs application with 15 namespaces in the cluster and run continuously. Prometheus/Alertmanager would be installed and managed by Prometheus Operator, Grafana would be installed via sample addon config yaml.
+
+Abnormal metrics breaking SLO and suspicious logs would be recorded in the alertmanager-webhook pod, corresponding notification would be pushed to slack channel #stability-test(Note: any slack webhook url uploaded to github would be deactivated, contact @richardwxn for latest webhook url to put in the alertmanagerconfig.yaml)
+
+If you want to run against a public release(stable or dev), specify the target release TAG/VERSION/RELEASE_URL, check more details about accepted argument at [install_readme](https://github.com/istio/tools/tree/master/perf/istio-install#setup-istio). You can specify the namespace number of the servicegraph workloads by setting NAMESPACE_NUM var.
+
+For example
+ 
+run with istio 1.5.4:
+
+`VERSION=1.5.4 NAMESPACE_NUM=15 ./long_running.sh`
+
+run with istio 1.6.5 prerelease:
+
+`RELEASE_URL=https://storage.googleapis.com/istio-prerelease/prerelease/1.6.5/istio-1.6.5-osx.tar.gz NAMESPACE_NUM=15  ./long_running.sh`
```

Comment:
how do I override hub / tag?
This will be needed if we want to test CVE builds and such.

Reply:
It accept same arguments as setup_istio.sh, so we can override hub/tag, etc.